libpod/runtime_migrate.go

```diff
@@ -0,0 +1,43 @@
+package libpod
+
+import (
+	"context"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+func (r *Runtime) migrate() error {
+	runningContainers, err := r.GetRunningContainers()
+	if err != nil {
+		return err
+	}
+
+	allCtrs, err := r.state.AllContainers()
+	if err != nil {
+		return err
+	}
+
+	logrus.Infof("stopping all containers")
+	for _, ctr := range runningContainers {
+		logrus.Infof("stopping %s", ctr.ID())
+		ctr.Stop()
+	}
+
+	for _, ctr := range allCtrs {
+		if strings.HasPrefix(ctr.config.ConmonPidFile, ctr.state.RunDir) {
```

Comment:
Can we try and match the full path, in case people were actually specifying a non-default path?

Reply:
sure, changing it